ValidateOtp/index.ts

```diff
@@ -0,0 +1,42 @@
+import * as express from "express";
+import * as winston from "winston";
+
+import { Context } from "@azure/functions";
+import { secureExpressApp } from "io-functions-commons/dist/src/utils/express";
+import { AzureContextTransport } from "io-functions-commons/dist/src/utils/logging";
+import { setAppContext } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import createAzureFunctionHandler from "io-functions-express/dist/src/createAzureFunctionsHandler";
+
+import { getConfigOrThrow } from "../utils/config";
+import { REDIS_CLIENT } from "../utils/redis";
+import { GetValidateOtp } from "./handler";
+
+const config = getConfigOrThrow();
+
+// tslint:disable-next-line: no-let
+let logger: Context["log"] | undefined;
+const contextTransport = new AzureContextTransport(() => logger, {
+  level: "debug"
+});
+winston.add(contextTransport);
+
+// Setup Express
+const app = express();
+secureExpressApp(app);
+
+// Add express route
+app.post(
+  "/api/v1/merchant/cgn/otp/:otpcode/validate",

```

Comment:
Do we need the merchant path? It seems redundant because we are inside the _cgn-merchant_ function.

Reply:
IMHO we need it since it is a new function and its' Info endpoint should be different from [this](https://github.com/pagopa/io-functions-cgn/blob/e2607c695556fecdccce8e969c5da978a641fc61/Info/function.json#L8)